src/util/invariant.cpp

```diff
@@ -93,6 +102,29 @@ void print_backtrace(
       out << '\n' << std::flush;
     }
 
+#elif defined(_WIN32)
+
+  void *stack[50];
+  HANDLE process = GetCurrentProcess();
+
+  SymInitialize(process, NULL, TRUE);
+
+  auto number_of_frames = CaptureStackBackTrace(0, 50, stack, NULL);
+
+  auto symbol = static_cast<SYMBOL_INFO *>(calloc(sizeof SYMBOL_INFO + 256, 1));
```

Comment:
This allocation procedure is mad enough you should probably cite https://docs.microsoft.com/en-us/windows/win32/api/dbghelp/ns-dbghelp-symbol_info for the future reader who thinks that surely can't be how you do it :)

Reply:
added